examples/online/online-tracker.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { createClient } = require('redis');
+
+/**
+ * Factory function to create an OnlineTracker instance
+ *
+ * @param {Object} redisClient - The Redis client instance.
+ * @returns {Object} An object with methods for user activity tracking.
+ */
+
+function createOnlineTracker(redisClient) {
+  // Method to add a user's activity
+  async function add(user) {
+    const now = Date.now();
+    try {
+      await redisClient.hSet('online_users', user, now.toString());
+    } catch (err) {
+      console.error('Error setting user activity:', err);
+    }
```

Comment:
Noticing now that you have both this *and* the call to `add` in a try/catch, which is unnecessary.